spec/support/vet360/stub_vet360.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+def stub_vet360(person = nil)
```

Comment:
This stub is to provide support for the data we are populating as part of the user serializer when other test classes are using it.